minetest.conf.example

```diff
@@ -3611,4 +3603,3 @@
 #    See https://github.com/minetest/irrlicht/blob/master/include/Keycodes.h
 #    type: key
 # keymap_decrease_viewing_range_min = -
-
```

Comment:
?

Reply:
that's my ide removing extra new lines at the end of the file automatically - the final \n is still there

Reply:
The script will readd it next time but :shrug: